rollup.config.js

```diff
@@ -0,0 +1,35 @@
+import babel from '@rollup/plugin-babel'
+import commonjs from '@rollup/plugin-commonjs'
+import resolve from '@rollup/plugin-node-resolve'
+import typescript from 'rollup-plugin-typescript2'
+
+import { DEFAULT_EXTENSIONS } from '@babel/core'
+const extensions = [...DEFAULT_EXTENSIONS, '.ts', '.tsx']
+
+const packageJson = require("./package.json")
+
+export default {
+	input: 'lib/index.ts',
+	output: [
+		{
+			file: packageJson.main,
+			format: "cjs",
+			sourcemap: true
+		},
+		{
+			file: packageJson.module,
+			format: "esm",
+			sourcemap: true
+		}
+	],
+	plugins: [
+		resolve({ extensions }),
+		typescript(),
+		commonjs({ extensions }),
+		babel({
```

Comment:
What babel config is it using?

Reply:
The existing one that is already in place from just using babel directly before this pr:
https://github.com/nextcloud/nextcloud-event-bus/blob/master/babel.config.js